src/circuit/circuit_builder.rs

```diff
@@ -18,8 +18,8 @@
 //!     // Add a source operator.
 //!     let source_stream = circuit.add_source(Generator::new(|| "Hello, world!".to_owned()));
 //!
-//!     // Add a sink operator and wire the source directly to it.
-//!     circuit.add_sink(
+//!     // Add a unary operator and wire the source directly to it.
+//!     circuit.add_unary_operator(
```

Comment:
so now inspect has an output?

Reply:
Inspect is special, all operators have outputs. 
With the new API it doesn't really matter, though.